### ISSUE TITLE
[feature]: Display curated measurement set names

### ISSUE THREAD

**Issue by BenJamesBen** [freesewing/freesewing#5898]
### What is it that you would like to see happen? 🤔

To make it easier to discuss patterns made using curated measurement sets, I would like it if each set had its own name or ID number shown to users. This would be a public-facing name or id, not necessarily the actual name of the model.

That way, I could say things like "To reproduce the problem, use Charlie with curated measurement set 'Delta`" or "I tested using curated measurement set # 302". Instead, right now I would have to say "using the 2nd person from the left" or "the guy with the orange shirt" which might be ambiguous as we add more measurement sets.

### Are you a FreeSewing patron? 😃

Yes, I am a tier-2 patron ❤️

### Additional context

I was able to mock up something with the below code change, to show id numbers.

```diff
--- a/sites/shared/components/curated-sets.mjs
+++ b/sites/shared/components/curated-sets.mjs
@@ -76,7 +76,16 @@ const SetLineup = ({ sets = [], href = false, onClick = false }) => (
       if (onClick) props.onClick = () => onClick(set)
       else if (typeof href === 'function') props.href = href(set.id)
 
-      if (onClick) return <button {...props} key={set.id}></button>
+      if (onClick) return (
+        <div>
+          <div>
+            <button {...props} key={set.id}></button>
+          </div>
+          <div>
+            <span class="text-2xl">&nbsp;&nbsp;&nbsp;&nbsp;#{set.id}</span>
+          </div>
+        </div>
+      )
       else if (href) return <Link {...props} key={set.id}></Link>
       else return <div {...props} key={set.id}></div>
     })}
```

![Screenshot 2024-01-25 at 9 22 10 AM](https://github.com/freesewing/freesewing/assets/109869956/c8225bb4-9620-4fdd-849b-24cd65b74737)

**Comment by joostdecock** [comment]
Problem I see with this is that people are listed from smoll to toll, so if tomorrow we add another person, it will change the IDs.

So I think I prefer to just use a name, because we already show the name of the curated set: https://freesewing.org/curated-sets
So, let's be consistent and use the name also in this view.

**Comment by BenJamesBen** [comment]
FYI, we do not currently show the name of curated sets, at least not to ordinary, non-privileged users. 

But, I'll look into changing the privilege-check or otherwise change the code to make it appear for non-privileged users.

No curated set names, as seen on https://freesewing.org/curated-sets :
![Screenshot 2024-02-14 at 9 53 17 AM](https://github.com/freesewing/freesewing/assets/109869956/d5615a39-43da-4532-bafb-d7514d023149)

**Comment by BenJamesBen** [comment]
Okay, now I see. On the Curated Measurements Sets page, if you click on the image, it epxands more info and shows the name of the measurement set. But, this only happens on that page. When viewing curated sets in the new pattern measurements-picker view, clicking on the image creates the new pattern and you never see the the name. 

I hadn't previously looked at the Curated Measurements page, so I wasn't aware that the names were available.

**Comment by BenJamesBen** [comment]
I accidentally and incorrectly linked #6075 to this Issue. If this Issue is accidentally closed due to #6075 getting merged, please re-open.
